demo/package.json

```diff
@@ -51,7 +51,8 @@
       "\\.js$": "<rootDir>/jest.preprocessor.js"
     },
     "setupFiles": [
-      "<rootDir>/jest.setup.js"
+      "<rootDir>/jest.setup.js",
+      "jest-prop-type-error"
```

Comment:
What does this do?

Reply:
Without that, proptype errors are ignored by jest.
Refs: https://github.com/tasitlabs/tasit/issues/60 (I think that change solves this issue).
Now, when test case faces a proptype error, it will fail.

Reply:
Cool. I'll resolve this thread.